### PR TITLE
[UIPO-54] Make Carousel work vertical

### PR DESCRIPTION
**Pull request by KhrystynaKovtun**
I also added Carousel to Product Image, styled Carousel in Quick view, styled Product Details, fixed some warnings.

### CODE REVIEW

File: src/components/Accordion/Accordion.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Accordion extends Component {
   dataMapper(array) {
     const { open } = this.props;
 
-    return array.map((element) => ({ ...element, isHidden: !open }));
+    return array.map((element) => ({ isHidden: !open, ...element }));
```
Comment (nadiiaLebedenko) [inline src/components/Accordion/Accordion.js:24]: Are you sure element.isHidden won't overwrite new value?

File: src/components/ProductDetailsPage/ProductDetailsPage.js [hunk 2]
```diff
@@ -1,28 +1,62 @@
...
+    const productImageData = images.map(({ claudinaryId }) => ({ src: `${process.env.IMAGE_URL}/${claudinaryId}` }));
+
+    const addToWishlistWithNotification = () => {
+      const { showMessage, addToWishlist } = props;
+      addToWishlist(id);
+      showMessage(ADDED_TO_WISHLIST_NOTIFICATION);
+    };
+
+    return (
+      <>
```
Comment (nadiiaLebedenko) [inline src/components/ProductDetailsPage/ProductDetailsPage.js:48]: This shoudn't be here

File: src/components/ProductDetailsPage/ProductDetailsPage.js [hunk 3]
```diff
@@ -1,28 +1,62 @@
...
+        <Button
+          className="go-back-btn"
+          icon="arrow left"
+          onClick={goBack}
+        >
+      Go Back
```
Comment (nadiiaLebedenko) [inline src/components/ProductDetailsPage/ProductDetailsPage.js]: Please, fix indents

**Review by nadiiaLebedenko — CHANGES_REQUESTED** [review]

File: src/components/Accordion/Accordion.js (hunk 1 above)
Comment (KhrystynaKovtun) [reply src/components/Accordion/Accordion.js:24]: That is why I did it.  if in data is present isHidden I want to set to that value, not a default one)

File: src/components/ProductDetailsPage/ProductDetailsPage.js (hunk 2 above)
Comment (KhrystynaKovtun) [reply src/components/ProductDetailsPage/ProductDetailsPage.js:48]: Why?

**Review by nadiiaLebedenko — APPROVED** [review]

**Pull request merged**